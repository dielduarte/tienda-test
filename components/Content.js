import styled from 'styled-components';

const Content = styled.section`
  flex: 2;
  height: 100vh;
  background-color: ${props => props.theme.lightGray};
  overflow-y: scroll;
  padding-bottom: 50px;
`;

export const ContentSpace = styled.div`
  padding: 0 20px;
`;

export default Content;
