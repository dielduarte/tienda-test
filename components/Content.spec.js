import { shallow } from 'enzyme';
import 'jest-styled-components';

import Content, { ContentSpace } from './Content';

describe('Content', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Content />);
    expect(wrapper).toMatchSnapshot();
  });

  it('ContentSpace', () => {
    const wrapper = shallow(<ContentSpace />);
    expect(wrapper).toMatchSnapshot();
  });
});
