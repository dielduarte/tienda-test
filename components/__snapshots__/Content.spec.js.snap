// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content ContentSpace 1`] = `
.c0 {
  padding: 0 20px;
}

<div
  className="c0"
/>
`;

exports[`Content should render correctly 1`] = `
.c0 {
  -webkit-flex: 2;
  -ms-flex: 2;
  flex: 2;
  height: 100vh;
  overflow-y: scroll;
  padding-bottom: 50px;
}

<section
  className="c0"
/>
`;
